frontend/trpc/routes/dividendComputations.ts

```
@@ -0,0 +1,267 @@
+    .input(
+      z.object({
+        companyId: z.string(),
+        totalAmountInUsd: z.number().positive().min(0.01, "Amount must be at least $0.01"),
+        dividendsIssuanceDate: z.string().date("Invalid date format"),
+        returnOfCapital: z.boolean().default(false),
+        investorReleaseForm: z.boolean().default(false),
+        investorDetails: z.string().optional(),
+      })
+    )
+    .mutation(async ({ ctx, input }) => {
+      console.log("dividendComputations.create called with input:", input);
```

ESLint

[error] Unexpected console statement.

(no-console)